diff --git a/src/stringify.js b/src/stringify.js
index bec2e587a7a2fd20bae08aecc7df0231d31a51d6..8548cce5b7dee97f3d198c6cb400979a8cd4e977 100644
--- a/src/stringify.js
+++ b/src/stringify.js
@@ -44,10 +44,6 @@ export function stringify(value, reducers) {

 	/** @param {any} thing */
 	function flatten(thing) {
-		if (typeof thing === 'function') {
-			throw new DevalueError(`Cannot stringify a function`, keys);
-		}
-
 		if (thing === undefined) return UNDEFINED;
 		if (Number.isNaN(thing)) return NAN;
 		if (thing === Infinity) return POSITIVE_INFINITY;
